common/src/main/java/tw/edu/ntut/sce/java18/common/service/RoomTypeService.java

```diff
@@ -0,0 +1,20 @@
+package tw.edu.ntut.sce.java18.common.service;
+
+import tw.edu.ntut.sce.java18.common.dao.RoomTypeDao;
+import tw.edu.ntut.sce.java18.common.dao.impl.RoomTypeDaoImpl_JDBC;
+import tw.edu.ntut.sce.java18.common.model.RoomTypeServiceBean;
+
+public class RoomTypeService {
+  RoomTypeDao dao;
+
+  public RoomTypeService() {
+    dao = new RoomTypeDaoImpl_JDBC();
+  }
+
+  public RoomTypeServiceBean queryRoomType(String type) {
+
+    var roomTypeBean = dao.queryRoomTypeInfo(type);
+    var roomType = new RoomTypeConverter().convert(roomTypeBean);
+    return roomType;
```

Comment:
Local variable `roomType` is redundant
```suggestion
    return new RoomTypeConverter().convert(roomTypeBean);
```